.github/workflows/cd-kpinetwork-backend.yml

```diff
@@ -58,6 +58,7 @@ jobs:
           TF_VAR_aws_account_id: ${{ secrets.AWS_ACCOUNT_ID }}
           TF_VAR_db_username: ${{ secrets.DB_USERNAME }}
           TF_VAR_db_password: ${{ secrets.DB_PASSWORD }}
+          TF_VAR_git_token: ${{ secrets.TOKEN }}
```

Comment:
You should change the name, the name isn't enough descriptive (maybe CODEBUILD_TOKEN is a better name)